### PR TITLE
Show info in admin settings about PHP opcache if disabled

### PR DESCRIPTION
**Pull request by MorrisJobke**
bildschirmfoto 2017-02-15 um 00 04 03

I tried really hard to write unit tests for this, but it's simply not possible to change those settings during runtime. 😢 

* [x] add this to documentation  https://github.com/nextcloud/documentation/pull/367

### CODE REVIEW

File: tests/Settings/Controller/CheckSetupControllerTest.php [hunk 1]
```diff
@@ -311,6 +311,14 @@ public function testCheck() {
 			->method('linkToDocs')
 			->with('admin-reverse-proxy')
 			->willReturn('reverse-proxy-doc-link');
+		$this->urlGenerator->expects($this->at(3))
+			->method('linkToDocs')
+			->with('admin-code-integrity')
+			->willReturn('http://doc.owncloud.org/server/go.php?to=admin-code-integrity');
```
Comment (rullzer) [inline tests/Settings/Controller/CheckSetupControllerTest.php]: tiny nitpick... but why not use our own domain ;)?

**Review by rullzer — APPROVED** [review]

File: tests/Settings/Controller/CheckSetupControllerTest.php (hunk 1 above)
Comment (MorrisJobke) [inline tests/Settings/Controller/CheckSetupControllerTest.php]: That's the other check I and I would like to keep them in sync with the code above :/

**Comment by MorrisJobke** [comment]
ping @nickvergessen @blizzz @schiessle

**Comment by MorrisJobke** [comment]
ping @nickvergessen @blizzz @schiessle

File: core/js/setupchecks.js [hunk 2]
```diff
@@ -148,6 +148,18 @@
...
 					}
+					if(!data.isOpcacheProperlySetup) {
+						messages.push({
+							msg: t(
+								'core',
+								'The PHP Opcache is not properly configured. <a target="_blank" rel="noreferrer" href="{docLink}">For better performance we recommend ↗</a> to use following settings in the <code>php.ini</code>:',
+								{
+									docLink: data.phpOpcacheDocumentation,
+								}
+							) + "<pre><code>opcache.enable=On\nopcache.enable_cli=1\nopcache.interned_strings_buffer=8\nopcache.max_accelerated_files=10000\nopcache.memory_consumption=128\nopcache.save_comments=1\nopcache.revalidate_freq=1</code></pre>",
```
Comment (nickvergessen) [inline core/js/setupchecks.js]: `opcache.enabled=On` and `opcache.enabled_cli=1`? Can you make both `1`?

Comment (MorrisJobke) [inline core/js/setupchecks.js]: No - this somehow doesn't work. `opcache.enabled` works here on Ubuntu 16.04 only with `On` ... all other options with `1`. I'm confused as well 😕

Comment (nickvergessen) [inline core/js/setupchecks.js]: http://php.net/manual/en/opcache.configuration.php really odd then:
```
Name 	Default 	Changeable 	Changelog
opcache.enable 	"1" 	PHP_INI_ALL 	 
```

**Review by nickvergessen — APPROVED** [review]

File: core/js/setupchecks.js (hunk 2 above)
Comment (MorrisJobke) [inline core/js/setupchecks.js]: > http://php.net/manual/en/opcache.configuration.php really odd then:

I know. But better be on the safe side for this.

**Comment by MorrisJobke** [comment]
Documentation: https://github.com/nextcloud/documentation/pull/367

**Pull request merged**